app/main.py

```diff
@@ -1,8 +1,38 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Place each argument on a new line in the `__init__` on the new line including `self`. It improves the code readability and helps with reading the type annotations. Change it here and in the `CarWashStation` class as well.